### DISCUSSION TITLE
Is it possible to fetch the entire json data from the json file, placed in the fixtures folder and paste it's content in the textarea?

### DISCUSSION THREAD

**Discussion by Pranesh21** [cypress-io/cypress#23845] [Questions and Help]
Is it possible to fetch the entire json data from the json file, placed in the fixtures folder and paste it's content in the textarea?

For exmple i have json in fixture file

{
"firstName": "Bidhan",
"lastName": "Chatterjee",
"age": 40,
"address":
{
"streetAddress": "144 J B Hazra Road",
"city": "Burdwan",
"state": "Paschimbanga",
"postalCode": "713102"
},
"phoneNumber":
[
{
"type": "personal",
"number": "09832209761"
},
{
"type": "fax",
"number": "91-342-2567692"
}
]
}

**I need to pass this entire json data in ui textbox is that possible?**

**Comment by Priya-LaunchIT** [comment]
```
it(`Test`, function () {
        var sometext
           cy.readFile('cypress\\fixtures\\dummy.json').then((data) => {
            sometext =JSON.stringify(data)
           cy.get('textarea').type(sometext,{parseSpecialCharSequences: false})
            })
        

    })
```
